cpg-language-python/src/test/kotlin/de/fraunhofer/aisec/cpg/frontends/python/statementHandler/StatementHandlerTest.kt

```diff
@@ -25,21 +25,42 @@
  */
 package de.fraunhofer.aisec.cpg.frontends.python.statementHandler
 
+import de.fraunhofer.aisec.cpg.TranslationResult
 import de.fraunhofer.aisec.cpg.frontends.python.PythonLanguage
 import de.fraunhofer.aisec.cpg.graph.*
+import de.fraunhofer.aisec.cpg.graph.statements.AssertStatement
+import de.fraunhofer.aisec.cpg.graph.statements.expressions.Literal
 import de.fraunhofer.aisec.cpg.test.analyze
 import de.fraunhofer.aisec.cpg.test.analyzeAndGetFirstTU
 import de.fraunhofer.aisec.cpg.test.assertResolvedType
 import java.nio.file.Path
 import kotlin.test.Test
 import kotlin.test.assertEquals
 import kotlin.test.assertNotNull
+import org.junit.jupiter.api.BeforeAll
+import org.junit.jupiter.api.TestInstance
 
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
 class StatementHandlerTest {
 
+    private lateinit var topLevel: Path
+    private lateinit var result: TranslationResult
+
+    @BeforeAll
+    fun setup() {
+        topLevel = Path.of("src", "test", "resources", "python")
+    }
+
+    fun analyzeFile(fileName: String) {
```

Comment:
Personally I don't like this pattern, but I won't block this PR any further. What is the advantage of storing the result in the class? I only see this pattern usually if multiple tests execute on the same file, but each test is executed on another file and the `result` property gets overwritten. This feels very error prone. I get that the tests contain a lot of boilerplate but then we should see how we can rewrite the `analyze` functions to be leaner and still return the result/tu instead of assigning it to a class member.

Reply:
I totally agree with you. I used this pattern for the tests in ArgumentsHandlerTest.kt, in which we are using the same file for every test. 
I will improve it over time.